js/GroupButton/Overview.md

```diff
@@ -1,18 +1,18 @@
 ---
 layout: post
-title: Overview
-description: overview
+title: Overview in JavaScript GroupButton Control | Syncfusion
```

Comment:
Overview of JavaScript GroupButton Control | Syncfusion

Reply:
Done